### ISSUE TITLE
BRPickerStyle 'language' does not seem to work

### ISSUE THREAD

**Issue by skuske** [91renb/BRPickerView#225]
According to the documentation, setting `language` to `nil` will force to follow the system language. However, when I display a picker with `BRDatePickerModeYM` mode, the months still show up in **English**, although the system is set to **German**.

Can you provide a working example so that the months will show up in the system language? Thanks!

```
    BRPickerStyle *customStyle = [[BRPickerStyle alloc]init];
    customStyle.language=nil; //Follow system language
```

**Comment by skuske** [comment]
Any update on this? How can I make the month names to appear in the system language?

**Comment by skuske** [comment]
Fixed in 2.7.7
